taf/IP/IP_Metadata.py

```diff
@@ -579,7 +579,7 @@ def plan_id_num(colname: str, alias: str):
         "PTNT_CNTL_NUM",
         "HLTH_CARE_ACQRD_COND_CD",
         "PTNT_STUS_CD",
-        "cast(BIRTH_WT_GRMS_QTY as numeric(12,3)) as BIRTH_WT_GRMS_QTY",
+        "cast(BIRTH_WT_GRMS_QTY as decimal(12,3)) as BIRTH_WT_GRMS_QTY",
```

Comment:
@FICU-tensileai  This data type change doesn't seem to be referenced in the ticket. Could you explain why this change was made?

Reply:
Good catch. Sorry. I was trying to debug one of the run failures and it was a suggestion from somewhere to see if that was causing it. It was not and I forgot to remove it. Removing it now.

Reply:
https://cms-dataconnect-val.cloud.databricks.com/?o=955724715920583#job/492738489039926/run/912910883953726
Removed and completed run to show it's still working without it.